.github/workflows/continuous-integration.yml

```diff
@@ -24,5 +32,5 @@ jobs:
           docker_registry_login: ${{ env.DOCKERHUB_USERNAME }}
           docker_registry_password: ${{ secrets.DOCKERHUB_PASSWORD }}
           docker_image_name: 'rasa/financial-demo'
-          docker_image_tag: 'latest'
-          rasa_sdk_version: '2.1.2'
+          docker_image_tag: ${{ env.TAG }}
+          rasa_sdk_version: '2.2.0'
```

Comment:
Not that we do this very much, but if we ever need to merge a PR into a branch that isn't master or 2.0, i think this will fail? maybe we should condition this whole step on the base branch being one of those branches

Reply:
Yup, the whole workflow is conditioned on that:

https://github.com/RasaHQ/financial-demo/blob/48dbb009d395ca337c77590464bda60df68872f9/.github/workflows/continuous-integration.yml#L4-L8